resources/plugins/test-carbon/webpack.config.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+const CamundaModelerWebpackPlugin = require('camunda-modeler-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  devtool: 'inline-source-map',
+  entry: './client/index.js',
+  output: {
+    path: require('path').join(__dirname, './dist'),
+    filename: 'client.js',
+  },
+  resolve: {
+    alias: {
```

Comment:
Let's move this to [camunda-modeler-webpack-plugin](https://github.com/camunda/camunda-modeler-webpack-plugin).

Or do you see any reason to wire manually in _every_ plug-in that uses this?